core/pom.xml

```diff
@@ -335,9 +335,63 @@
   <build>
     <outputDirectory>target/scala-${scala.binary.version}/classes</outputDirectory>
     <testOutputDirectory>target/scala-${scala.binary.version}/test-classes</testOutputDirectory>
+    <resources>
+      <resource>
```

Comment:
I believe we don't need this section since it automatically picks up anything under the resources directory.